library/src/test/java/com/novoda/downloadmanager/UnlinkedDataRemoverTest.java

```diff
@@ -0,0 +1,72 @@
+package com.novoda.downloadmanager;
+
+import java.util.ArrayList;
+import java.util.Arrays;
+import java.util.List;
+
+import org.junit.Test;
+
+import static com.google.common.truth.Truth.assertThat;
+
+public class UnlinkedDataRemoverTest {
+    private static final String FIRST_FILE = "a filename";
+    private static final String SECOND_FILE = "another filename";
+    private static final String THIRD_FILE = "yet another filename"; // Orphan
+
+    @Test
+    public void givenUnlinkedFilesInLocalStorage_whenRemoving_thenUnlinkedFilesAreDeleted() throws Exception {
+        // Arrange
```

Comment:
Would remove comments in favour of extracting more explanatory methods. This project doesn't follow the `given_when_then` structure but prefers `action_whenSomethingOccurs` see [here](https://github.com/novoda/download-manager/blob/epic_download_manager_v2/library/src/test/java/com/novoda/downloadmanager/CallbackThrottleByTimeTest.java)